src/views/DashboardView.vue

```diff
@@ -76,18 +76,14 @@ import Spinner from '../components/Spinner.vue'
 import { useBoardsNewStore } from '../stores/boardsNew'
 import { ref, toRefs, computed } from 'vue'
 import { useWindowSize } from '@vueuse/core'
+import { useUserStore } from '../stores/user'
 
 const isLoading = ref(true)
 
 const { boards, currentBoard, boardColumns, isConfirmationPopupShown } = toRefs(
   useBoardsNewStore()
 )
-const { getBoardsData } = useBoardsNewStore()
-
-;(async () => {
-  await getBoardsData()
-  isLoading.value = false
-})()
+const userStore = useUserStore()
```

Comment:
I dislike using storeToRefs in scope of a component as it's easy to loose context (makes code harder to follow). Assigning store to variable makes it simple across whole codebase. I'd use storeToRefs in case I need one or two values from the store in scope of a small function, but despite that function store is not needed in whole component.

Example:
```ts
// Dummy.vue
const userStore = useUserStore();
// first usage
const someValue = ref("");
const otherValue = computed(() => userStore.user.name.toLowerCase());
// some functions taking another 20 or more lines

async function onSubmit() {
  await fetch('some-url', {method: "POST", body: JSON.stringify(
    { 
      username: userStore.user.name, // explicitly indicate this value comes from the store
      age: otherValue.value, // this one doesn't :)
    }
)})
}
```

Imagine the same code with pure refs, would it be clear which comes from the store and which doesn't?